provider/image_test.go

```diff
@@ -428,12 +428,6 @@ func TestConfigureDockerClient(t *testing.T) {
 		assert.NoError(t, err)
 		assert.Equal(t, expected, actual.DaemonHost())
 	})
-	t.Run("Given a host passed via environment, a client should be configured", func(t *testing.T) {
```

Comment:
Is there a reason this is removed?

Reply:
This test never worked correctly. All it verified was that given zero input to the config, a client would still be generated. It was doing so by using the default Docker host on the system (there's a [test below that verifies this](https://github.com/pulumi/pulumi-docker/pull/672/files#diff-b8cb2a07f6e7e3d68c17c754cde490ecaa0bc627272f3dc0d8c9705bc7fe9a3bL527), and this test erroneously clai med that it was generating a client via environment variables.